doma-spring-boot-samples/doma-spring-boot-sample-docker-compose/src/test/java/org/seasar/doma/boot/sample/ApplicationTest.java

```diff
@@ -6,15 +6,15 @@
 
 @SpringBootTest
 @TestPropertySource(properties = {
-		"spring.datasource.url=jdbc:h2:mem:test;DB_CLOSE_DELAY=-1",
-		"spring.datasource.username=sa",
-		"spring.datasource.password=",
-		"spring.datasource.driver-class-name=org.h2.Driver",
-		"doma.dialect=H2"
+    "spring.datasource.url=jdbc:h2:mem:test;DB_CLOSE_DELAY=-1",
```

Comment:
Use docker compose in tests as well. That means PostgreSQL should be used in tests instead of H2.

By default, Spring Boot’s Docker Compose support is disabled when running tests.

To enable Docker Compose support in tests, set `spring.docker.compose.skip.in-tests` to `false`.